app/factories/flood_risk_engine/steps/form_object_factory.rb

```diff
@@ -4,11 +4,12 @@ class FormObjectFactory
       class << self
         def form_object_for(step, enrollment)
           klass = form_object_class_map.fetch(step.to_sym) do
-            fail "No form object defined for step #{step}"
```

Comment:
`fail` is used a lot in the existing code, but personally I have no problem using `raise` instead.